filebeat/input/default-inputs/inputs.go

```diff
@@ -32,5 +33,7 @@ func Init(info beat.Info, log *logp.Logger, components beater.StateStore) []v2.P
 }
 
 func genericInputs() []v2.Plugin {
-	return []v2.Plugin{}
+	return []v2.Plugin{
+		unix.Plugin(),
```

Comment:
Adding unix socket to genericInputs that are supported on all platforms.

Newer windows versions do support AF_UNIX, but only streaming unix sockets AFAIK

Reply:
What does that mean from support standpoint? Filebeat supports older Windowses, e.g. Windows Server 2012, which cannot handle AF_UNIX based on my quick internet research. How are we informing users about the limitation of this input?

Reply:
We already link the unix input into Filebeat as is. So it is no real change. Removing it from windows build would be a breaking change.

On older windows versions the input will fail to create the listener. That's all